SB2018/2018-04-03.md

```diff
@@ -0,0 +1,13 @@
+---
+to: All Teams
+subject: Media Consent Forms
+---
+
+Hi all,
+
+In order for anyone to attend the competition, a 'Media Consent' Form must be filled out. If students are over 16, they can fill out the forms themselves; however, if students are under 16, the form must be signed by their parent/guardian.
+
+These forms must be filled out by all attendees and brought to the competition in order to gain entry.
+Any additional guests can sign a form on the door as long as they're 16 or over.
+
+It's important that the form is correctly filled out to ensure the line isn't held up at reception on the Saturday morning.
```

Comment:
Perhaps phrase this in a manner which is clearer that we'd like the teachers to actually do something?
> In order to ensure we can process arrivals smoothly, please ensure that your team has all completed Media Consent Forms before you arrive at the competition. If you are brining any under 16s, please ensure that their parents or guardians have completed the proper sections as they may be denied entry if the forms are incorrect.

Reply:
fixed in 8f9e501dc2fd51aeb106b0311df66208e53ffa1f